conda_recipe/test_install_conda_packages.py

```diff
@@ -0,0 +1,37 @@
+#! /bin/python
+
+import argparse
+from pathlib import Path
+from subprocess import Popen, PIPE
+from sys import stderr
+import tarfile
+
+
+def main():
```

Comment:
This test is great! It would be good to put this in the CI; otherwise, it will have a tendency to rot.

I'm wondering if these files might make sense under `tests/` or at least outside of `conda_recipe/`. I think if there is a common pattern running tests, even if we might skip some depending on some configuration, it will be one less thing to remember.

Reply:
Should I recreate the top-level `tests` directory? We killed it because it contained just Python tests before. But a top-level `tests` would make sense for packaging tests like that and for formatting checkers, since we run them as tests in CI.

I'll look into running this in CI as a dependent job after the conda packaging job. I suspect we don't want to make it required though, since conda packaging still takes an insanely long time in CI (45 minutes).

Reply:
I'm moving them to tests. In part because we have had several people say "How are you all locally running tests?" and a "tests" directory might help with that.